benchmark/benchmarks.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -eu
+
+readonly SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
```

Comment:
```suggestion
readonly SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
```

Just in case there is a space in the parent path.

Reply:
Thanks! fixed :smile: